development/architecture/migration-guide/validation.md

```diff
@@ -7,12 +7,12 @@ weight: 70
 
 The purpose of validation is to protect the application from failures, state inconsistency, security related issues etc. PrestaShop has few different levels of validation:
 * **Forms** (the [CRUD]({{<relref "/8/development/architecture/migration-guide/forms/crud-forms">}}) or [Settings]({{<relref "/8/development/architecture/migration-guide/forms/settings-forms">}})) - are using [Symfony validation constraints](https://symfony.com/doc/4.4/validation.html#constraints) to prevent the user from providing invalid information while also enriching the user experience. For example if a form input expects `Yes` or `No`, but user submitted value `XYZ` we want to reject the data and inform the user what was wrong. The best approach (if possible) is to prevent user providing the wrong value in the first place, so in this example we could simply use a checkbox or select input with only 2 possible selections. There are some custom constraints implemented in PrestaShop [here](https://github.com/PrestaShop/PrestaShop/tree/1.7.8.x/src/Core/ConstraintValidator).
```

Comment:
```suggestion
* **Forms** (the [CRUD]({{<relref "/8/development/architecture/migration-guide/forms/crud-forms">}}) or [Settings]({{<relref "/8/development/architecture/migration-guide/forms/settings-forms">}})) - are using [Symfony validation constraints](https://symfony.com/doc/4.4/validation.html#constraints) to prevent the user from providing invalid information while also enriching the user experience. For example if a form input expects `Yes` or `No`, but user submitted value `XYZ` we want to reject the data and inform the user what was wrong. The best approach (if possible) is to prevent user providing the wrong value in the first place, so in this example we could simply use a checkbox or select input with only 2 possible selections. There are some custom constraints implemented in PrestaShop [here](https://github.com/PrestaShop/PrestaShop/tree/8.0.x/src/Core/ConstraintValidator).
```